src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/controllers/web/RoleController.kt

```diff
@@ -0,0 +1,34 @@
+package org.vaccineimpact.orderlyweb.controllers.web
+
+import org.vaccineimpact.orderlyweb.ActionContext
+import org.vaccineimpact.orderlyweb.controllers.Controller
+import org.vaccineimpact.orderlyweb.db.OrderlyUserRepository
+import org.vaccineimpact.orderlyweb.db.UserRepository
+import org.vaccineimpact.orderlyweb.viewmodels.RoleViewModel
+
+class RoleController(context: ActionContext,
+                     private val userRepo: UserRepository) : Controller(context)
+{
+    constructor(context: ActionContext) : this(context, OrderlyUserRepository())
+
+    fun getGlobalReportReaders(): List<RoleViewModel>
+    {
+        val users = userRepo.getGlobalReportReaderRoles()
```

Comment:
Not sure UserRepository is the right name for this if it's giving us roles as well as users.

Reply:
:+1: Split into 2 repos: https://github.com/vimc/orderly-web/pull/110/commits/4e26c838c2f1aad1dbbae2dd2428fb3e9a3f5b61